server.go

```diff
@@ -24,5 +24,5 @@ func main() {
 
 	n := negroni.Classic()
 	n.UseHandler(router)
-	n.Run(":3000")
+	n.Run(":3001")
```

Comment:
I'll bump the frontend up to 4000. I think that would make it easy to differentiate overall.